.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
don't include IDE files

Reply:
added the file name in .gitignore

Reply:
same

Reply:
removed from repository in commit number fa9ddf7